08-collab.md

```diff
@@ -97,3 +97,12 @@ Fast-forward
  1 file changed, 1 insertion(+)
  create mode 100644 pluto.txt
 ~~~
+
+> ## Review changes {.challenge}
+>
+> The Owner makes some changes in the repository without giving any information
+> to the Collaborator. How can the Collaborator find out what has changed with 
+> command line? And on GitHub? 
+> 
+> The Collaborator makes some local changes and committed them. Has the process
+> to review changes changed? 
```

Comment:
I'm not sure if I understand :) You mean the Collaborator didn't push the new commits to GitHub, so the Owner can not get the changes? Or you just mean there is no difference on who pushes the commit because both have push permissions on the repo? Sorry if I'm missing something.